docs/trex_getdata.md

```diff
@@ -72,15 +73,53 @@ After you have a worksheet object, you can call one of the methods to access the
 
 ## Get summary data from a worksheet
 
+Starting with Tableau 2022.4 and the Dashboard Extensions API library v1.10, you should use the `getSummaryDataReaderAsync()` method to get data from a worksheet. This method returns a `DataTableReader` object that you can use to access the data.
+
+If your summary data is less than 4,000,000 rows (400 pages), you can use the `DataTableReader.getAllPagesAsync()` method to retrieve a single `DataTable` from the `DataTableReader`. The following example shows how to do this.
+
+```javascript
+
+// Use `await` only inside an `async` method
+
+let vizActiveSheet = viz.workbook.activeSheet;
+if (vizActiveSheet.sheetType === "worksheet") {
+  const dataTableReader = await vizActiveSheet.getSummaryDataReaderAsync();
+  const dataTable = await dataTableReader.getAllPagesAsync();
+  await dataTableReader.releaseAsync();
+  // ... process data table ...
+}
+
+```
+
+If your summary data contains more than 4,000,000 rows (or 400 pages), use the summary `DataTableReader` to iterate through the pages of data for all rows in the worksheet. In this case, you process each `DataTable` sequentially. You can control the page size, using the optional `pageRowCount` parameter when you call `getSummaryDataReaderAsync`. The default page size is 10,000 rows. Using the `DataTableReader` you can create a loop to retrieve each page of summary data, using `DataTableReader.pageCount` or `DataTableReader.totalRowCount` property to determine the number of pages to process. Use the `getPageAsync` method to get the `DataTable` from each page. After you have retrieved all pages of the summary data, call the `releaseAsync` method to free up memory from the `DataTableReader`.
+
 ```javascript
 
+// Use `await` only inside an `async` method
+
+let vizActiveSheet = viz.workbook.activeSheet;
+if (vizActiveSheet.sheetType === "worksheet") {
+    const dataTableReader = await vizActiveSheet.getSummaryDataReaderAsync();
```

Comment:
Same as above. We don't have the viz.workbook.activiteSheet in extensions.

Reply:
Thanks! I missed that.

Reply:
I got rid of the `viz.workbook.activeSheet`, but forgot to remove the `vizActiveSheet` from the remaining line. 
Done now.

Reply:
I don't think you have committed that yet. It still has the vizActiveSheet in my review.

Reply:
I guess I spoke too soon (before the commit/push). It should be good now. 
https://github.com/tableau/extensions-api/pull/520/commits/88f444fb7b27aa4680cd7bb2c7d77628f470ef5c

Reply:
Strange. That commit had a banner that said it wasn't connected to anything.  I'm running this through a fork, which might be the problem.  I did another push (git push upstream). That seems to have worked.

Reply:
I had one more commit to fix some formatting with a catch block.